text/0010-keptn-cli-enhancements.md

```diff
@@ -0,0 +1,62 @@
+# keptn cli and API enhancements
+
+Enhance keptn-cli with the ability to list and get details of onboarded projects and services.
+
+## Motivation
+
+Right now when I'm using keptn within a CI/CD Pipeline I want to check if my service is already onboarded in keptn, if not it should be done automatically.
+From Ops-Perspective I want to get a list of all projects which are using keptn. Like in helm a keptn list command to retrieve a list of all projects and for more details a keptn get/details <project> command which returns all services within the project would be benificial.
+As workaround I'm checking if in my kubernetes cluster a lighthouse-configmap is existing, but this is not a pretty solution.
+
+## Explanation
+
+```bash
+# keptn list
+Project             Onboarded                       Git Repository
+sockshop            2020-02-03 11:50                https://xyz/project.git
+```
+
+```bash
+# keptn get sockshop
+Project: sockshop
+Onboarded: 2020-02-03 11:50
+Git Repository: https://xyz/project.git
+Services:
+- carts-service
+- payment-service
+```
+
+```bash
+# keptn get sockshop service carts-service
+Project: sockshop
+Service: carts-service
+Last artifact: myimage:v1.0.2
```

Comment:
Details like

* Last artifact
* last deployment
* etc...

should be listed per stage